src/precompile.jl

```diff
@@ -14,3 +14,17 @@ precompile(run_and_collect, (Cmd,))
 precompile(cudaconvert, (Function,))
 precompile(Core.kwfunc(cudacall), (NamedTuple{(:threads, :blocks), Tuple{Int64, Int64}},typeof(cudacall),CuFunction,Type{Tuple{}}))
 precompile(Core.kwfunc(launch), (NamedTuple{(:threads, :blocks), Tuple{Int64, Int64}},typeof(launch),CuFunction))
+
+using PrecompileTools: @setup_workload, @compile_workload
+@static if VERSION >= v"1.11.0-DEV.1603"
```

Comment:
I don't recall why this is only `v"1.11.0-DEV.1603"` it should be useful in anycase.